includes/themes.php

```diff
@@ -43,6 +43,11 @@ function __construct() {
 			require_once WPBDP_PATH . 'includes/admin/controllers/class-themes-admin.php';
 			$this->admin = new WPBDP_Themes_Admin( $this, wpbdp()->licensing );
 		}
+
+		$theme_requires = $this->get_active_theme_data( 'requires' );
```

Comment:
This function is already quite long. Can we move this new code to a new function?